deepchem/metalearning/tests/test_maml_reload_torch.py

```diff
@@ -0,0 +1,90 @@
+"""Test that MAML models can be reloaded."""
+
+import deepchem as dc
+import numpy as np
+import pytest
+from deepchem.metalearning.torch_maml import MetaLearner, MAML
```

Comment:
Add this line within the try-except block as it is also indirectly importing torch.

Reply:
Yes
Jax tests are passing now.